Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:alpine as BUILD
+
+WORKDIR /relayer
+
+# Copy the files from host
+COPY . .
+
+# Update and install needed deps prioir to installing the binary.
+RUN apk update && \
+    apk --no-cache add make=4.2.1-r2 git=2.24.3-r0 && \
+    make install
+
+FROM alpine:edge
+
+ENV RELAYER /relayer
+
+RUN addgroup rlyuser && \
```

Comment:
Why are we setting up a user account here? Shouldn't docker provide the isolation we want?

Reply:
```
When a Dockerfile doesn’t specify a USER, it defaults to executing the container using the root user. In practice, there are very few reasons why the container should have root privileges. Docker defaults to running containers using the root user. When that namespace is then mapped to the root user in the running container, it means that the container potentially has root access on the Docker host. Having an application on the container run with the root user further broadens the attack surface and enables an easy path to privilege escalation if the application itself is vulnerable to exploitation.
```

I follow this when dealing with docker containers that will get used in a non-testing env. I will be updating the gaia image to use a user as well. Its an added level of security, buts it's up to you if you'd like it or not

Reply:
I don't have a strong opinion here, but prefer simplicity to additional code.